AcademicReward/Logic/GroupLogic.cs

```diff
@@ -0,0 +1,68 @@
+﻿using AcademicReward.Database;
+using AcademicReward.ModelClass;
+using AcademicReward.Resources;
+
+namespace AcademicReward.Logic
+{
+    /// <summary>
+    /// Primary Author: Maximilian Patterson
+    /// Secondary Author: 
+    /// Reviewer: 
+    /// </summary>
+    public class GroupLogic : ILogic
+    {
+        private IDatabase GroupDB;
+
+        /// <summary>
+        /// LoginGroupLogic constructor
+        /// </summary>
+        public GroupLogic()
```

Comment:
Please update comments, also, are we going to show anything to the user if an error occurs? We really shouldn't always be returning that no error occurred.